recipe-client-addon/lib/PreferenceExperiments.jsm

```diff
@@ -306,9 +310,10 @@ this.PreferenceExperiments = {
       throw new Error(`Invalid value for preferenceBranchType: ${preferenceBranchType}`);
     }
 
-    if (!experimentType.startsWith("normandy-pref")) {
+    if (experimentType.length > MAX_EXPERIMENT_SUBTYPE_LENGTH) {
       throw new Error(
-        `Expected experimentType to begin with "normandy-pref". Found "${experimentType}".`
+        `experimentType must be less than ${MAX_EXPERIMENT_SUBTYPE_LENGTH} characters. ` +
+        `"${experimentType}" is ${experimentType.length} long.`
```

Comment:
Nit: missing a unit - 'characters' ? :-)

Reply:
I think that since the previous line gives the unit (characters) it is fine to not repeat it.